lib/view_component/preview.rb

```diff
@@ -42,21 +42,11 @@ def render_args(example, params: {})
         result.merge(layout: @layout)
       end
 
-      # Returns the component object class associated to the preview.
-      def component
```

Comment:
Could this be being used by apps using previews? 🤔

Reply:
@juanmanuelramallo Can you have a look at this PR? There was some unused/untested preview code.

Reply:
Just looked over the production code of previews and it's indeed not being used. Safe to remove